src/bfx_tools/topiary.ml

```diff
@@ -0,0 +1,198 @@
+open Biokepi_run_environment
+open Common
+
+type output_type = [
+  | `HTML of string
+  | `CSV of string
+]
+
+type predictor_type = [
+  | `NetMHC
```

Comment:
It's scary to see these all enumerated in a wrapper API, these are likely to change in the future (especially the ones that call to IEDB).